src/master/develop/management.md

```diff
@@ -250,11 +264,218 @@ If you're using another development language, please refer to the [immugw](/mast
+
+Note: Replication settings take effect without the need of reloading the database.
+
+Following example shows how to update database using `UpdateDatabaseV2` method.
+
+:::: tabs
+
+::: tab Go
+
+```go
+	client, err := immudb.NewClient()
+	if err != nil {
+    	log.Fatal(err)
+	}
+
+	ctx := context.Background()
+	
+	err = client.OpenSession(ctx, []byte(`immudb`), []byte(`immudb`), "defaultdb")
+	if err != nil {
+    	log.Fatal(err)
+	}
+
+	defer client.CloseSession(ctx)
+
+	res, err := client.UpdateDatabaseV2(ctx, "mydb", &schema.DatabaseNullableSettings{
+		TxLogCacheSize: &schema.NullableUint32{Value: 1000},
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+```
+:::
+
+::: tab Java
+```java
+	List<String> dbs = immuClient.databases();
+	// List of database names
+```
+:::
+
+::: tab Python
+This feature is not yet supported or not documented.
+Do you want to make a feature request or help out? Open an issue on [Python sdk github project](https://github.com/codenotary/immudb-py/issues/new)
+:::
+
+::: tab Node.js
+This feature is not yet supported or not documented.
+Do you want to make a feature request or help out? Open an issue on [Node.js sdk github project](https://github.com/codenotary/immudb-node/issues/new)
+:::
+
+::: tab .Net
+This feature is not yet supported or not documented.
+Do you want to make a feature request or help out? Open an issue on [.Net sdk github project](https://github.com/codenotary/immudb4dotnet/issues/new)
+:::
+
+::: tab Others
+If you're using another development language, please refer to the [immugw](/master/immugw/) option.
+:::
+
+::::
+
 <br/>
 
-## Index Cleaning
+## Index cleaning
 
-It's important to keep disk usage under control. `CleanIndex` it's a temporary solution to launch an internal clean routine that could free disk space.
+Maintaining a healthy disk usage is crucial. immudb has two operations operations aiming to remove unreferenced data from the index.
+A full index clean-up is achieved by calling `CompactIndex`, which is a routine that creates a fresh index based on the current state, removing all intermediate data generated over time. The index is generated asynchronous, so new transactions may take place as it is created. As a result, if the server is constantly overloaded, there will likely be blocking times when the newly compacted index replaces the current one.
+In the case of continuous load on the server, the `FlushIndex` operation may be used instead. It will dump the current index into disk while partly removing unreferenced data. The `cleanupPercentage` attribute indicates how much space will be released. Even though this operation blocks transaction processing, choosing a small percentage e.g. 0.1 may not significantly hinder normal operations while reducing used storage space. 
```

Comment:
```suggestion
In the case of continuous load on the server, the `FlushIndex` operation may be used instead. It will dump the current index into disk while partly removing unreferenced data. The `cleanupPercentage` attribute indicates how much space will be scanned for unreferenced data. Even though this operation blocks transaction processing, choosing a small percentage e.g. 0.1 may not significantly hinder normal operations while reducing used storage space. 
```